dashboard/src/components/Properties.js

```diff
@@ -1,34 +1,57 @@
 // @flow
 
 import React, { Component } from "react";
+import { Button, Glyphicon } from "react-bootstrap";
 
 import "../assets/css/Properties.css";
 
 class Properties extends Component {
     render() {
-        let props = JSON.parse(this.props.currentNode),
+        let props = this.props.currentNode !== "{}" &&
+            JSON.parse(this.props.currentNode.title),
```

Comment:
what is the purpose of `props` variable? also the name might not be the best because readers might confuse it with `this.props`